src/style.css

```diff
@@ -1,3 +1,72 @@
 body {
   margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: normal;
+  --activeLinkColor: #00acdc;
+  --simpleLinkColor: #000;
+}
+
+.nav__list {
+  list-style: none;
+  display: flex;
+  align-items: center;
+  padding: 11px 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+}
+
+.nav__link {
+  text-transform: uppercase;
+  color: var(--simpleLinkColor);
+  text-decoration: none;
+  cursor: pointer;
+  position: relative;
+  height: 100%;
+  padding: 23px 0;
+  margin-right: 20px;
+}
+/* i don't understand why it doesnt't work
+As i understand,i override only the last element.
+Why margin-right 0 assigned for all links?
+The specificity is higher in nav__link:last-child */
+.nav__link:last-child {
+  margin-right: 0;
+}
```

Comment:
you should add indents to nav_item class, because now in each block nav_link is technically last because it's the only one
<img width="488" alt="Знімок екрана 2023-07-27 о 11 32 24" src="https://github.com/mate-academy/layout_moyo-header/assets/106310796/3db23acb-17bc-4830-b1de-76e3889fca24">